zulipterminal/helper.py

```diff
@@ -646,13 +647,16 @@ def check_narrow_and_notify(
     outer_narrow: List[Any], inner_narrow: List[Any], controller: Any
 ) -> None:
     current_narrow = controller.model.narrow
-
     if (
         current_narrow != []
         and current_narrow != outer_narrow
         and current_narrow != inner_narrow
     ):
-        controller.report_success("Message is sent outside of current narrow.")
+        key = primary_key_for_command("NARROW_MESSAGE_RECIPIENT")
+
+        controller.report_success(
+            f"Message is sent outside of current narrow. Press [{key}] to narrow to conversation."
```

Comment:
If we keep this message this long, I'd welcome this being displayed a little longer (eg. the 6 we used in the other PR).

However, my remaining query here is whether we want to customize this logic/message a little further:
- in all-messages narrow, we never see this, but may still want to go to the conversation
- in stream (or all-PM) narrows which hold the message, we may also still want to go to the conversation
- if a message is outside the narrow, we always want to show this

To address these combinations we may want to split this messaging change out into a followup commit and/or PR, since the functionality itself will be present and consistent with the help menu and documentation already in the rest of this commit. This is an extra hint, and it'd be good to handle it well.